### PR TITLE
Update vmw-how-vls-install-usage-meter-4.md

### PR DESCRIPTION
**Pull request by mikepukcloud**
Changed commands around password expiry to correct format

### CODE REVIEW

File: articles/vmware/vmw-how-vls-install-usage-meter-4.md [hunk 1]
```diff
@@ -107,13 +107,13 @@ To disable password timeout:
...
 
-2. Run the commands `chage -M 0 user-usagemeter` and `chage -M 0 user-umauditor`.
+2. Run the commands `chage -M 0 root` , `chage -M 0 usagemeter` and `chage -M 0 umauditor`.
 
 To increase the timeout value: 
 
 1. Using the VM's console, log in as `root`.
 
-2. Run the commands `chage -M <expiration-in-days> user-usagemeter` and `chage -M <expiration-in-days> user-umauditor`.
+2. Run the commands `chage M <expiration-in-days> root` , `chage -M <expiration-in-days> usagemeter` and `chage -M <expiration-in-days> umauditor`.
```
Comment (suehighmoor) [inline articles/vmware/vmw-how-vls-install-usage-meter-4.md:116]: @mikepukcloud, just checking, but should there be a - in front of the first M?

**Review by SteveHallUKCLoud — APPROVED** [review]
LGTM other than Sue's comment.

**Comment by mikepukcloud** [comment]
OMG how did i do that i coped and pasted and everything, but yeah your right there should be a -

**Review by suehighmoor — APPROVED** [review]
LGTM

**Pull request merged**